src/pretix/base/email.py

```diff
@@ -241,6 +242,8 @@ def get_email_context(**kwargs):
             kwargs['invoice_address'] = kwargs['order'].invoice_address
         except InvoiceAddress.DoesNotExist:
             kwargs['invoice_address'] = InvoiceAddress()
+        finally:
+            kwargs.setdefault("position_or_address", kwargs['invoice_address'])
```

Comment:
The changes in this file makes the already existing `{name_*}` placeholders available always when there's an order, because we can always use the orders invoice address as a fallback for lookup (although it might be set to an empty one).